writer_test.go

```diff
@@ -550,6 +550,93 @@ func TestNewMasterPlaylistWithAlternatives(t *testing.T) {
 	if m.ver != 4 {
 		t.Fatalf("Expected version 4, actual, %d", m.ver)
 	}
+	fmt.Printf("%v\n", m)
+}
+
+// Create new master playlist supporting closed-caption=none
+func TestNewMasterPlaylistWithClosedCaptionEqNone(t *testing.T) {
```

Comment:
This is a very comprehensive test, but I don't think it really needs to be. Can the same writing test be achieved by just having a single master playlist, with one variant setting the Alternative Closed Captions to none?

And then the test can be a straight forward strings.Contains, such as https://github.com/grafov/m3u8/blob/f55aa7a0ae5bfe63e82ee5adefa3aa4e8692b3dd/writer_test.go#L192-L221 ?